src/js/plugin/sdk/stores/PipelineStore.js

```diff
@@ -0,0 +1,44 @@
+
+export const FILTERS_PIPELINE = {};
```

Comment:
Sure! The idea is to let only the `PipelineStore` modify the internal state of the pipelines. Currently exporting this variable makes it possible to other components of the core UI do modify it (that's what I'm currently doing in the test cases) but this is not true for plugins, this is because plugins cannot import the module directly, all that plugins see are the objects exported to its `pluginWindow.marathonPluginInterface`, so no plugin could ever reach the original exported `FILTERS_PIPELINE`, right? Does this make sense?